Dockerfile.ubi

```diff
@@ -29,11 +29,20 @@ RUN microdnf install -y \
         make \
         sqlite-devel \
         tar \
-        zlib-static
+        zlib-static \
+        autoconf \
+        automake \
+        libtool \
+        which \
+        dejagnu \
+        bison \
+        flex \
+        perl
```

Comment:
It is fine to leave for now, but my plan with `Dockerfile.ubi` is to be as close as possible to what OpenShift will ship in the future. Are there any other plans with this Dockerfile?
The thing is that installing build tools and building stuff from scratch is a bit of a no-no for OpenShift.
Anyway, this is totally fine for this PR, just a food for thought (and something I wanted to discuss on our Tuesday meeting).

Reply:
@jhrozek I was actually torn between fixing the errors I had supporting AppArmor for the OpenShift building process by a) leveraging conditional build and removing it completely or b) building libapparmor from source. I don't really have a good answer as to why I chose the latter. :smile: 

Users in RHEL/CentOS/Fedora will tend to use SELinux and users in Debian/Ubuntu/etc will tend to use AppArmor. Realistically I don't think we need to support both simultanebously, apart from cases in which a cluster have node pools that go across both spectrum.

For the time being I will just remove AppArmor from the OpenShift build through conditional build. If the need/use case arise in the future we can always retrofit.